jax/experimental/stax.py

```diff
@@ -49,8 +50,7 @@ def Dense(out_dim, W_init=glorot_normal(), b_init=normal()):
   def init_fun(rng, input_shape):
     output_shape = input_shape[:-1] + (out_dim,)
     k1, k2 = random.split(rng)
-    b_shape = (1,) * (len(output_shape) - 1) + (out_dim,)
-    W, b = W_init(k1, (input_shape[-1], out_dim)), b_init(k2, b_shape)
+    W, b = W_init(k1, (input_shape[-1], out_dim)), b_init(k2, (out_dim,))
```

Comment:
Could you please only revert this change, but keep the rest? It appears that this is the only part that breaks them

Reply:
Done.